.github/workflows/env.yml

```
@@ -0,0 +1,19 @@
+        required: false
+        default: test
+jobs:
+  print:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment-name }}
+    env:
+      USERNAME: ${{ vars.USERNAME }}
+      PASSWORD: ${{ secrets.PASSWORD }}
+    steps:
+      - run: echo "Hello, $USERNAME!"
+      - run: echo "Your password is $PASSWORD."
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)